spec/classes/init_spec.rb

```diff
@@ -16,6 +16,16 @@
         it { is_expected.to contain_service('mumble-server') }
         it { is_expected.to contain_user('mumble-server') }
       end
+
+      context 'with a server_password' do
+        let(:params) do
+          {
+            server_password: 'It is Secret: $(rm -r /)'
```

Comment:
huhooo, nice value for a test!  :+1: 

Can you update acceptance tests by adding this too ?
It looks we only test the general entrance password but not the supervisor one.

Reply:
Testing in progress… I do not have the tooling to check the acceptance tests on my current machine… may need some fixing :-)  I'll keep an eye on it and attempt to fix failures.

Reply:
Seems good now